util/PostgresMigrations/Scripts/2022-09-12_00_PasswordlessAuthRequests.psql

```diff
@@ -1,107 +1,5 @@
-
-ALTER TABLE "ProviderOrganization" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Provider" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Provider" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Policy" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Policy" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "OrganizationUser" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "OrganizationUser" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "OrganizationSponsorship" ALTER COLUMN "ValidUntil" TYPE timestamp with time zone;
-
-ALTER TABLE "OrganizationSponsorship" ALTER COLUMN "LastSyncDate" TYPE timestamp with time zone;
-
-ALTER TABLE "OrganizationApiKey" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Organization" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Organization" ALTER COLUMN "OwnersNotifiedOfAutoscaling" TYPE timestamp with time zone;
-
-ALTER TABLE "Organization" ALTER COLUMN "ExpirationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Organization" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Installation" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Group" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Group" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Grant" ALTER COLUMN "ExpirationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Grant" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Grant" ALTER COLUMN "ConsumedDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Folder" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Folder" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Event" ALTER COLUMN "Date" TYPE timestamp with time zone;
-
-ALTER TABLE "EmergencyAccess" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "EmergencyAccess" ALTER COLUMN "RecoveryInitiatedDate" TYPE timestamp with time zone;
-
-ALTER TABLE "EmergencyAccess" ALTER COLUMN "LastNotificationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "EmergencyAccess" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Device" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Device" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Collection" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Collection" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Cipher" ALTER COLUMN "RevisionDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Cipher" ALTER COLUMN "DeletedDate" TYPE timestamp with time zone;
-
-ALTER TABLE "Cipher" ALTER COLUMN "CreationDate" TYPE timestamp with time zone;
```

Comment:
For my own understanding, why was timezone added to the dates and then removed? Was it a scope change?

Reply:
Thank you for brining this up. I was not expecting these changes to appear in the migration - and was confused when they did. The only expected database changes for EF with this work were to add the AuthRequest table. I manually removed these column alterations, but meant to come back to it later to figure out what was going on.

I took a look at this again this morning and it turns out this is expected for the first migration ran using version [6.0](https://www.npgsql.org/efcore/release-notes/6.0.html?tabs=annotations) or higher.  We've upgraded to that package version recently, but I don't think we've run a migration since then, so it appeared here.

I went ahead and added these lines back on [Add datetime conversions back to postgres migration script](https://github.com/bitwarden/server/pull/2276/commits/1a68476366e13cbf6f9f8c36c49c7f305555e86d)

Reply:
Makes sense, thanks for the explanation!